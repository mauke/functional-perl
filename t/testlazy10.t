#!/usr/bin/env perl

# Copyright (c) 2015 Christian Jaeger, copying@christianjaeger.ch
# This is free software. See the file COPYING.md that came bundled
# with this file.

use strict; use warnings; use warnings FATAL => 'uninitialized';

our $len= 759;
require "./meta/test.pl";

use Test::More;

$ENV{N}= 10;
$ENV{T}= 0;
$ENV{TZ}= "MET";
is readin ("functional_XML/testlazy|"), readin ("< t/testlazy10.expected");

done_testing;
