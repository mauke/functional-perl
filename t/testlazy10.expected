<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html lang="en" xml:lang="en" xmlns="http://www.w3.org/1999/xhtml"><head><title>example page - Hello World</title></head><body><h1>Hello World</h1><p>Garçon méchanique, "1 &lt; 2" is true. <a href="&quot;1 &lt; 2&quot;">this will be 404</a></p><table border="1"><tr><td>1</td><td>one</td></tr><tr><td>2</td><td>two</td></tr><tr><td>3</td><td>three</td></tr></table><p>10</p><p>9</p><p>8</p><p>7</p><p>6</p><p>5</p><p>4</p><p>3</p><p>2</p><p>1</p><p>0</p><hr/><p>By <a href="http://christianjaeger.ch">Christian Jaeger</a>, last modified at Thu Jan  1 00:00:00 1970 (or something).</p></body></html>
